### DISCUSSION TITLE
Is it possible to show lines changed, like in GitHub?

### DISCUSSION THREAD

**Discussion by Azzurite** [gitextensions/gitextensions#10015] [Q&A]
GitHub always shows this nice little summary of how many lines were changed in a particular commit: 
![image](https://user-images.githubusercontent.com/2777158/170802750-fdd8429d-6639-4df9-93ee-676e535b82c4.png)

Is it possible to show something like this in Git Extensions, too? I couldn't immediately find any setting, but maybe I'm missing something.

**Comment by RussKie** [comment]
Could you please elaborate how this information is useful?
What git command would you run to get this info?
Where would you display it?

Do note, that every command has costs (execution, parsing, display, etc.), and this will likely have perf implications.
